README.md

```diff
@@ -24,26 +24,23 @@ Parameters
 | argument | purpose | required/optional                                    |
 | ------- | ------- | ------------------------------------------ |
 | --r1_in | Path(s) to FASTQ(s) containing read 1   | required                |
-| --r1_out | Path to output FASTQ 1   | required                |
 | --r2_in | Path(s) to FASTQ(s) containing read 2   | optional              |
-| --r2_out | Path to output FASTQ 2   | optional              |
 | --r3_in | Path(s) to FASTQ(s) containing UMIs for paired end with non-inline UMIs | optional              |
 | --pattern1 | pattern or regex for extracting UMIs in read 1   | optional              |
 | --pattern2 | pattern or regex for extracting UMIs in read 2   | optional              |
-| --prefix | the prefix used for statistics output | optional              |
+| --prefix | Specifies the start of the output files | required              |
 | --data | paired or single end sequencing   | required              |
 | --separator | String separating the UMI sequence in the read name   | required              |
 | --umilist | Path to file with valid UMIs | optional              |
 | --inline | UMIs inline with reads or not | optional              |
```

Comment:
This has been switched to sub commands `inline` and `separate`. Again, I'm not sure if you want to keep it that way.

Reply:
I will add the sub commands to make the python version fully compliant